### PR TITLE
Images performance boost + placeholder

### PR DESCRIPTION
**Pull request by ArielBlum**

### CODE REVIEW

**Comment by zakiva** [comment]
status of this PR?

**Comment by ArielBlum** [comment]
Ready for merge

File: app/src/main/java/com/example/zakiva/santa/Helpers/Infra.java [hunk 1]
```diff
@@ -317,9 +321,25 @@ public static void getWinnersFromFirebase(final Context context) {
...
+                int size = HallOfFame.dataHashWinners.size();
+                ArrayList<String[]> items = new ArrayList<String[]>();
+
+                for (HashMap hm : HallOfFame.dataHashWinners.values()) {
+                    String[] item = new String[2];
+                    item[0] = (String) hm.get("imageUrl");
+                    item[1] = (String) hm.get("minusKey").toString();
+                    items.add(item);
+                }
+                Collections.sort(items, new java.util.Comparator<String[]>() {
```
Comment (zakiva) [inline app/src/main/java/com/example/zakiva/santa/Helpers/Infra.java:334]: what's the diff between this sort and the sort in the activity above?

Comment (ArielBlum) [reply app/src/main/java/com/example/zakiva/santa/Helpers/Infra.java:334]: This sort is for preloading the images (it makes sure we load only the first HallOfFamePreDownloadLimit images). The sort in the activity is for showing *all* the images in the right order.

**Pull request merged**